tests/end-to-end/backup-restore-test/README.md

```diff
@@ -0,0 +1,128 @@
+# End-to-end Backup and Restore Tests
+
+## Overview
+
+This project contains end-to-end backup and restore tests for the Kyma installation. The tests run daily on Prow to validate if the backup and restore process works for all components.
```

Comment:
I think its running once a week on friday

Reply:
good point, I will update it.

Reply:
Done